test/e2e/common/setup-kubestellar.sh

```diff
@@ -22,6 +22,10 @@ if [[ "$KFLEX_DISABLE_CHATTY" = true ]] ; then
    echo "disable_chatty_status = $disable_chatty_status"
 fi
 
+SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+COMMON_SRCS="${SRC_DIR}/../common"
+source "$COMMON_SRCS/setup-shell.sh"
```

Comment:
Why is this stuff here? I do not see any use of it later in this script (which is not `source`d, so the environment modifications have not effect on anything once this script exits).

Reply:
Doesn't the setup-shell.sh contains the wait-for-cmd that setup-kubestellar.sh uses?

Reply:
Oh, right, this PR execs this script on its own.